clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/EcsCloudProvider.java

```diff
@@ -6,7 +6,7 @@
 import java.lang.annotation.Annotation;
 
 @Component
-class EcsCloudProvider implements CloudProvider {
+public class EcsCloudProvider implements CloudProvider {
```

Comment:
Good one

Reply:
We should make a PR against the main project, with this change.  We'll need this change elsewhere as well

Reply:
We can just sneak in with the first change that needs it. It will be evident the first time you try to use the class

Reply:
I made this PR already, so no need to make it 👍